source/patterns/@aws-solutions-constructs/core/lib/sagemaker-defaults.ts

```diff
@@ -11,15 +11,58 @@
  *  and limitations under the License.
  */
 
-import { CfnNotebookInstanceProps } from '@aws-cdk/aws-sagemaker';
+import {
+  CfnNotebookInstanceProps,
+  CfnModelProps,
+  CfnModel,
+  CfnEndpointConfigProps,
+  CfnEndpointProps,
+} from '@aws-cdk/aws-sagemaker';
 
-export function DefaultSagemakerNotebookProps(roleArn: string, kmsKeyId: string,
-  subnetId?: string, securityGroupIds?: string[]): CfnNotebookInstanceProps {
+export function DefaultSagemakerNotebookProps(
+  roleArn: string,
+  kmsKeyId: string,
+  subnetId?: string,
+  securityGroupIds?: string[]
```

Comment:
The pattern used by constructs is that defaults return a fixed list of defaults. Callers then user overrideProps to adjust those fixed props to their needs.